src/main/webapp/resources/js/pages/launch/components/ParametersModal/SaveParametersAsFooter.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
File comments

Reply:
Updated in da2f590075a4125763860023854e9d25c38ac202